Lombiq.Hosting.Tenants.Management/Manifest.cs

```diff
@@ -21,7 +21,7 @@
 [assembly: Feature(
     Id = HideRecipesFromSetup,
     Name = "Lombiq Hosting - Tenants Management - Hide Recipes From Setup",
-    Description = "Ability to configure recipe categories witch won't be available during tenant setup.",
+    Description = "Ability to configure recipe tags that won't be available during tenant setup.",
```

Comment:
Talán így kicsit érthetőbb:
```suggestion
    Description = "Adds the ability to hide recipes from the setup screen based on configurable tags.",
```